### PR TITLE
Create a plot gallery

### PR DESCRIPTION
**Pull request by CKehl**
Creating a Lagrangian plot gallery to share plotting and visualisation examples from the group with the Parcels community.

Outcome of the Open Workspace Meeting (OWM).

Contact persons: @LauraGomezNavarro  @CKehl

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -783,6 +786,32 @@ <h5>Support for unstructured grids</h5>
     </div>
   </div>
 
+  <hr>
+  <h2 id="gallery">Gallery</h2>
+
+  <h3 id="bathymetry">Bathymetric</h3>
```
Comment (erikvansebille) [inline index.html:792]: Not sure this title is needed right now. When we get to a very large set we can think again about dividing in categories, but for now let's not do that?
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -74,14 +74,17 @@
           <a class="nav-link" href="#tutorials">Tutorials</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link" href="#developmentstatus">Development status</a>
+          <a class="nav-link" href="#developmentstatus">Development</a>
         </li>
         <li class="nav-item">
           <a class="nav-link" href="faq.html">Frequently Asked Questions</a>
```
Comment (erikvansebille) [inline index.html:80]: Also need to abbreviate to FAQ to fit in the header?
```suggestion
          <a class="nav-link" href="faq.html">FAQ</a>
```

**Review by erikvansebille — COMMENTED** [review]
Very nice idea! A few suggestions in the review

File: index.html (hunk 1 above)
Comment (CKehl) [reply index.html]: agreed. We can categorize it later. In today's OWM, we just created this one, first example, and I made the binder connection (thanks to @willirath - took your parcels binder as a template) so we can see the first result. Next steps are to populate the gallery repo and then we can come up with a structure when we have a better sample and impression of the kind of gallery objects.

File: index.html (hunk 2 above)
Comment (CKehl) [reply index.html:80]: that would help indeed - when I checked locally, the navbar was a bit off and the change in 'Development' was sufficient. But if it's okay for you to make the shift to FAQ, then that would be great for the navbar.

**Pull request merged**